sentences/ro/_common.yaml

```diff
@@ -1,12 +1,12 @@
 language: ro
 responses:
   errors:
-    no_intent: "Scuze, nu am înțeles cererea"
-    no_area: "Nu există nici o arie numită {{ area }}"
-    no_domain: "Aria {{ area }} nu conține {{ domain }}"
-    no_device_class: "Aria {{ area }} nu conține {{ device_class }}"
-    no_entity: "Niciun dispozitiv sau entitate cu numele {{ entity }}"
-    handle_error: "A apărut o eroare în timpul procesării cererii"
+    no_intent: "Îmi pare rău, nu am înțeles. Poți, te rog, să repeți?"
+    no_area: "Îmi pare rău, nu este nici o cameră numită {{ area }}"
```

Comment:
Area nu inseamna camera. Poti avea arii de tipul curte, parcare, beci etc., care nu sunt camere. E adevarat ca "arie" suna nasol, dar as inlocui cu "zona", nu cu foarte specificul "camera"

Reply:
Da, ai dreptate, este ok cu ”zona”. Cred totuși că dacă punem la răspunsuri în față expresii precum ”îmi pare rău”, ”scuze” îi dă o notă mai naturală răspunsului și nu pare chiar robotic. Eu folosesc de vreo jumătate de an Rhasspy și mi-am făcut pentru răspunsuri niște scripturi în care am pus mai multe variante sinonime de răspuns și la fiecare solicitare dă alt răspuns. Am văzut că în formatul în care trebuie să ne încadrăm se acceptă doar o singură variantă de răspuns pus între ghilimele. Mă gândeam dacă am putea să punem mai multe variante de răspuns și cu random. Eu am ceva cam așa făcută o automatizare : ["ok","oof ce lene ai","cât de greu poate să fie,","să știi că este ultima dată,"] | random}}, am aprins lumina din {area}. Cu asta am vrut să dea niște răspunsuri amuzante dar se pot pune mai multe variante de răspunsuri decente.